### PR TITLE
Azure Edge Updates and Edge Connectivity Statuses initial commit

### PR DESCRIPTION
**Pull request by roopavidhya**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/identity/Azure.Identity.BrokeredAuthentication/api/Azure.Identity.BrokeredAuthentication.net462.cs [hunk 1]
```diff
@@ -3,10 +3,12 @@ namespace Azure.Identity.BrokeredAuthentication
     public partial class InteractiveBrowserCredentialBrokerOptions : Azure.Identity.InteractiveBrowserCredentialOptions
     {
         public InteractiveBrowserCredentialBrokerOptions(System.IntPtr parentWindowHandle) { }
+        public bool? IsMsaPassthroughEnabled { get { throw null; } set { } }
```
Comment (jsquire) [inline sdk/identity/Azure.Identity.BrokeredAuthentication/api/Azure.Identity.BrokeredAuthentication.net462.cs:6]: This file should not be included in this set of changes.  Please rebase.

**Review by jsquire — DISMISSED** [review]

File: sdk/identity/Azure.Identity.BrokeredAuthentication/api/Azure.Identity.BrokeredAuthentication.net462.cs (hunk 1 above)
Comment (roopavidhya) [reply sdk/identity/Azure.Identity.BrokeredAuthentication/api/Azure.Identity.BrokeredAuthentication.net462.cs]: This has been reverted, thanks !

File: sdk/edgeupdates/Azure.ResourceManager.EdgeUpdates/src/autorest.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: Edge
+namespace: Azure.ResourceManager.Edge
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs-pr/bad587f7237e08e93cd45e95988f8a95b30c23c1/specification/edge/updates/resource-manager/readme.md?token=GHSAT0AAAAAACCFCF22OBZDHAHA24F5YGAGZFXXEFQ
```
Comment (live1206) [inline sdk/edgeupdates/Azure.ResourceManager.EdgeUpdates/src/autorest.md:10]: This repo is for public SDK generation, can't point to private spec repo.

Comment (roopavidhya) [reply sdk/edgeupdates/Azure.ResourceManager.EdgeUpdates/src/autorest.md]: The token has been removed, thanks ! The swagger PR for this sdk is still in review, I will update the public branch as soon as the PR is merged.

**Comment by ArthurMa1978** [comment]
@roopavidhya this is new, did you get the namesapce review approval?

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Need namesapce review approval

**Comment by roopavidhya** [comment]
@ArthurMa1978, I will be abandoning this PR and raising a new one. Thanks

**Pull request closed without merging**